qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -58,6 +59,8 @@ def change_virtio_media(cdrom_virtio):
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
 
+    virtio_nic_mac = vm.virtnet[1].mac
```

Comment:
This script run by different variants and it seems not all tests have the second nic, would you double check if it will fail other cases with this line?

Reply:
Hi @vivianQizhu 

For this script, it will only run with all_drivers, it means the VM will have an extra virtio nic.